worker/src/main/scala/com/lucidchart/piezo/TriggerHistoryModel.scala

```diff
@@ -20,7 +19,7 @@ class TriggerHistoryModel(props: Properties) {
   val logger = LoggerFactory.getLogger(this.getClass)
   val connectionProvider = new ConnectionProvider(props)
 
-  def addTrigger(trigger: Trigger, actualStart: Option[Date], misfire: Boolean, fireInstanceId: Option[String]): Unit = {
+  def addTrigger(triggerName: String, triggerGroup: String, triggerFireTime: Option[Date], actualStart: Option[Date], misfire: Boolean, fireInstanceId: Option[String]): Unit = {
```

Comment:
what is the motivation for changing this from taking a trigger to taking a bunch of individual values, that all come from the trigger?

Reply:
It was just simpler to remove quartz dependencies in the model. Made writing the tests so much easier.

Reply:
it was harder to use trigger, since Trigger was just an interface.